builder/builder.go

```diff
@@ -101,10 +98,25 @@ func NewBuilder(sk *bls.SecretKey, ds flashbotsextra.IDatabaseService, relay IRe
 }
 
 func (b *Builder) Start() error {
+	// Start regular payload attributes updates
+	go func() {
+		c := make(chan types.BuilderPayloadAttributes)
+		go b.beaconClient.SubscribeToPayloadAttributesEvents(c)
+		for {
+			select {
+			case <-b.stop:
+				break
+			default:
+				payloadAttributes := <-c
```

Comment:
this should be a concurrent select
```
			select {
			case <-b.stop:
				break
			case payloadAttributes <-c:
                                b.OnPayloadAttribute(&payloadAttributes)
```

otherwise it can get indefinitely stuck waiting for payload attributes